solutions/swb-reference/integration-tests/tests/isolated/sshKeys/get.test.ts

```diff
@@ -3,60 +3,43 @@
  *  SPDX-License-Identifier: Apache-2.0
  */
 
-import ClientSession from '../../../support/clientSession';
-import Setup from '../../../support/setup';
+import { PaabHelper, PaabResources } from '../../../support/complex/paabHelper';
 import HttpError from '../../../support/utils/HttpError';
-import RandomTextGenerator from '../../../support/utils/randomTextGenerator';
 import { checkHttpError } from '../../../support/utils/utilities';
 
 describe('Get Key Pair negative tests', () => {
-  const setup: Setup = new Setup();
-  let adminSession: ClientSession;
-  let project: { id: string };
-  const randomTextGenerator = new RandomTextGenerator(setup.getSettings().get('runId'));
+  const paabHelper = new PaabHelper();
+  let paabResources: PaabResources;
 
   beforeEach(async () => {
-    adminSession = await setup.getDefaultAdminSession(); //TODO: change to getPA1Session()
-    const { data: costCenter } = await adminSession.resources.costCenters.create({
-      name: randomTextGenerator.getFakeText('fakeCostCenterName'),
-      accountId: setup.getSettings().get('defaultHostingAccountId'),
-      description: 'a test object'
-    });
-
-    const { data } = await adminSession.resources.projects.create({
-      name: randomTextGenerator.getFakeText('fakeCostCenterName'),
-      description: 'Project for list users for project tests',
-      costCenterId: costCenter.id
-    });
-
-    project = data;
+    paabResources = await paabHelper.createResources();
```

Comment:
I'd extract the resources here like
```
{
      adminSession,
      pa1Session,
      pa2Session,
      rs1Session,
      project1,
      project2
} = await paabHelper.createResources();
```
with the resources defined as `let` variables outside of the `beforeEach` so all the tests have access to them.

Reply:
good point